os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -473,6 +477,15 @@ trwifi_result_e wifi_netmgr_utils_deinit(struct netdev *dev)
 	if (ret == RTK_STATUS_SUCCESS) {
 		g_mode = RTK_WIFI_NONE;
 		wuret = TRWIFI_SUCCESS;
+		if (saved_scan_list) {

```

Comment:
do we need to hold mutex to maintain the integrity of cleaning saved_scan_list here?
tw_mutex_get(&scanlistbusy);

Reply:
Done as suggested.